src/ln/channelmanager.rs

```diff
@@ -992,7 +989,7 @@ impl ChannelManager {
 			cltv_expiry_delta: CLTV_EXPIRY_DELTA,
 			htlc_minimum_msat: chan.get_our_htlc_minimum_msat(),
 			fee_base_msat: chan.get_our_fee_base_msat(&*self.fee_estimator),
-			fee_proportional_millionths: self.fee_proportional_millionths,
+			fee_proportional_millionths: self.configuration.channel_options.fee_proportional_millionths,
```

Comment:
This needs to be the channel's local copy, no? Can we rename ChannelManager::configuration to something like default_new_channel_config to make it more obvious that this isnt the right one to use?